bpf-gpl/ut/nat_encap_test.c

```diff
@@ -21,5 +21,13 @@
 
 static CALI_BPF_INLINE int calico_unittest_entry (struct __sk_buff *skb)
 {
-	return vxlan_v4_encap(skb, HOST_IP, 0x02020202);
+	struct cali_tc_ctx ctx = {
+		.state = state_get(),
+		.skb = skb,
+		.fwd = {
+			.res = TC_ACT_UNSPEC,
+			.reason = CALI_REASON_UNKNOWN,
+		},
+	};
```

Comment:
Check `.state` here, if it's really needed for the next call?

Reply:
Doesn't look like it (and the verifier will tell us if we start using it so I'll remove for now).